ci-operator/step-registry/openshift/edge-tooling/lvms-ci/doctor/openshift-edge-tooling-lvms-ci-doctor-commands.sh

```
@@ -102,6 +102,9 @@ SRC_DIR="${EDGE_TOOLING_DIR}"
 PLUGIN_DIR="${SRC_DIR}/plugins/lvms-ci"
 cd "${SRC_DIR}"
 
+# Use the Prow bot OAuth token for GitHub API access (read-only, public repos)
+{ set +x; export GITHUB_TOKEN="$(cat /etc/github/oauth 2>/dev/null || true)"; set -x; }
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)